packages/compiler/src/template_parser/binding_parser.ts

```diff
@@ -244,6 +244,10 @@ export class BindingParser {
       targetProps: ParsedProperty[], keySpan?: ParseSourceSpan) {
     if (isAnimationLabel(name)) {
       name = name.substring(1);
+      if (keySpan) {
```

Comment:
```suggestion
      if (keySpan !== undefined) {
```
nit: this still needs the explicit check